forge-gui/res/cardsfolder/upcoming/dont_move.txt

```diff
@@ -0,0 +1,8 @@
+Name:Don't Move
+ManaCost:3 W W 
+Types:Sorcery
+A:SP$ DestroyAll | ValidCards$ Creature.tapped | SubAbility$ DBEffect | SpellDescription$ Destroy all tapped creatures. 
+SVar:DBEffect:DB$ Effect | Triggers$ TrigTaps | Duration$ UntilYourNextTurn | SpellDescription$ Until your next turn, whenever a creature becomes tapped, destroy it.
+SVar:TrigTaps:Mode$ Taps | ValidCard$ Creature | Execute$ TrigDestroy | TriggerDescription$ Until your next turn, whenever a creature becomes tapped, destroy it.
```

Comment:
Maybe `TriggerDescription$ Whenever a creature becomes tapped, destroy it.`

This might be just quibbling though